code/game/objects/items/storage/fancy.dm

```diff
@@ -146,6 +146,9 @@
 /obj/item/storage/fancy/cigarettes/examine(mob/user)
 	. = ..()
 	. += "<span class='notice'>Alt-click to extract contents.</span>"
+	var/obj/item/lighter/L = locate(/obj/item/lighter) in contents
+	if(L && contents.len > 0)
```

Comment:
```suggestion
	if(L)
```
The second condition is always true if the first condition is true.

Reply:
Empty lists are truthy afaik? either way just use length()

Reply:
Yea empty lists are truthy

Reply:
nvm i thought L and contents were the same list if(L) is good